data_structures/linked_list/doubly_linked_list.c

```diff
@@ -0,0 +1,264 @@
+/**
+ * @file    doubly_linked_list.c
```

Comment:
```suggestion
 * @file
```

Reply:
Fixed up, thank you! :slightly_smiling_face: